mysql-test/suite/innodb/t/instant_alter_stored_column_order.test

```diff
@@ -0,0 +1,18 @@
+# PS-9222 Testing regression introduced by e6e13a8f, which causes engine crashed
+
+--echo # PS-9222 - Testing if ALGORITHM=INSTANT crashes server
+CREATE TABLE t1 (c1 TINYTEXT COLLATE ascii_bin NOT NULL, c2 DATETIME(3) NOT NULL, c3 TEXT, PRIMARY KEY (c1(30)));
+INSERT INTO t1 (c1, c2, c3) VALUE ('k1','2021-12-21','something');
+INSERT INTO t1 (c1, c2, c3) VALUE ('k3','2021-12-21','something else');
+
+ALTER TABLE t1 ADD COLUMN c4 VARCHAR(18) NOT NULL, ALGORITHM=INSTANT;
```

Comment:
it looks like there is no re-ordering here? :thinking: it is a simple add column at the end.  there should not be any reorder in this case. 

@VarunNagaraju can you print the reorder array and figure out which fields are marked as reordered?

Reply:
related discussion here: https://perconadev.atlassian.net/browse/PS-9222?focusedCommentId=413537